handlers/bin/command-not-found

```diff
@@ -130,6 +131,5 @@ if __name__ == "__main__":
     try:
         main()
     except Exception:
-        pass
-
-    sys.exit(EX_NOTFOUND)
+        traceback.print_exc()
+        sys.exit(EX_NOTFOUND)
```

Comment:
There is a slight logic change here - if no exception is thrown, it will now exit 0, but as far as I can see, the other codepaths have their own exit anyway.
And a `exit 0` is probably also the right thing to do in absense of exceptions.

Would be nice to see some such reasoning in the commit message.

Reply:
I wrote this a while ago, but I believe my reasoning was that on success, the utility should return as such. Looking back on it a year later, I don't really know what I was thinking when I did it. I'll go ahead and fix that now.

Reply:
Hey, just wanna follow up on this. Again, happy to add the change here, but I don't know that there's actually a logic change as much as a semantic difference. Just want to make sure I'm making the right fix before I do.